index.js

```diff
@@ -11,7 +11,9 @@ const express = require('express'),
 
 const routeController = require('./controllers/routes/routes');
 
-const dburl = 'mongodb://localhost/chirpper'; 
+
+const dburl = 'mongodb://simonxcode:simonxcode1@ds143511.mlab.com:43511/chirpper'; 
```

Comment:
Oh no, we didn't want this on Github lol

Reply:
Correct. I was not thinking on this one. Thanks for the correct. 
Would changes in this editor(which I just made) be okay or should PR be deleted and redone?